zbus/tests/xml.rs

```diff
@@ -0,0 +1,11 @@
+#![cfg(feature = "quick-xml")]
+
+use std::convert::TryFrom;
+
+use zbus::quick_xml::Node;
+
+#[test]
+fn invalid_arg_type() {
+    let input = include_str!("data/invalid_arg_type.xml");
+    assert!(Node::try_from(input).is_err());
```

Comment:
even better if you test for a specific error using `matches`.

Reply:
ok